### PR TITLE
291

### PR DESCRIPTION
**Pull request by rkzel**
FE done. Left one const piece of data inside, until backend is connected...

### CODE REVIEW

File: apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js [hunk 1]
```diff
@@ -1,21 +1,157 @@
...
+          <UserLink>
+            <img src={application.user.avatar} style={{ width: '32px', height: '32px', marginRight: '10px'}} />
+            <SafeLink
+              href={url}
+              target="_blank"
+              style={{ textDecoration: 'none', color: '#21AAE7', marginRight: '6px' }}
+            >
+              {application.user.name ? application.user.name : application.user.login}
+            </SafeLink>
+            applied 8 days ago
```
Comment (Quazia) [inline apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js]: Can we get this as a variable even if it's hardcoded for the time being?

**Comment by Quazia** [comment]
> Can you merge dev into this branch and make sure all of the panels are accessible in this branch?

Nvm, seeing the context menu now. Everything looks good except the magic number hardcoding for the "applied 8 days ago".

File: apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js (hunk 1 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js]: done.

File: apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js [hunk 2]
```diff
@@ -72,7 +74,7 @@ class ReviewApplication extends React.Component {
             >
               {application.user.name ? application.user.name : application.user.login}
             </SafeLink>
-            applied 8 days ago
+            applied {etaAgo} ago
```
Comment (Quazia) [inline apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js]: The eta should be the estimated time of completion that's different from when it's submitted or when they apply right?

File: apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js [hunk 3]
```diff
@@ -1,21 +1,162 @@
...
+import { FormField, FieldTitle } from '../../Form'
+import { IconGitHub, CheckButton } from '../../Shared'
+
+// external data, all of it
+const application = {
+  user: {
+    login: 'rkzel',
+    name: 'RKZ',
+    avatar: 'https://avatars0.githubusercontent.com/u/34452131?v=4',
+    url: "https://github.com/rkzel"
```
Comment (Quazia) [inline apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js]: Double quotes are making linting fail.

**Pull request merged**